Description: Disable debugger tests
 which depend on unpackaged debugger_test crate, and uses the "cdb" aka
 Microsoft Console Debugger, which is, obviously, Windows only.
Forwarded: not-needed
Last-Update: 2025-11-25
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53 +53 @@
-debugger_visualizer = ["alloc"]
+#debugger_visualizer = ["alloc"]
@@ -61 +61 @@
-[[test]]
+[[disabled.test]]
@@ -67 +67 @@
-[dev-dependencies.debugger_test]
+[disabled.dev-dependencies.debugger_test]
@@ -70 +70 @@
-[dev-dependencies.debugger_test_parser]
+[disabled.dev-dependencies.debugger_test_parser]
--- a/tests/debugger_visualizer.rs
+++ b/tests/debugger_visualizer.rs
@@ -0,0 +1 @@
+#![cfg(feature = "debugger_test")]
